### PR TITLE
LPS 180710

### PR DESCRIPTION
**Pull request by robertoDiaz**
See the reproduction steps in https://issues.liferay.com/browse/LPS-180710

It seems that retrieving the asset entires from collections was missing, so the asset publisher didn't notify to subscribers.

I've created a new method, _getAssetListEntryAssetEntries and its logic is partially based in the code used to retrieve the elements in asset publisher by calling _com.liferay.asset.publisher.web.internal.display.context.AssetPublisherDisplayContext#getAssetEntryQuery_.

 I simplify the use and call _assetListAssetEntryProvider.getAssetEntries directly instead of creating a new asset entry query.

If you prefer to use the asset entry query, please tell me and I'll reformat it

**Suggestions for testing:**  modify this line: com/liferay/asset/publisher/web/internal/scheduler/CheckAssetEntrySchedulerJobConfiguration.java:77 to 

`return TriggerConfiguration.createTriggerConfiguration(1, TimeUnit.MINUTE);
`

So the cron is executed more often (default value is 24 hours) :D

If you need more context, please tell me!

Thank you in advance!

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by robertoDiaz** [comment]
[com.liferay.asset.publisher.web.internal.scheduler.helper.test.AssetEntriesCheckerHelperTest](https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/427702/testReport/com.liferay.asset.publisher.web.internal.scheduler.helper.test/AssetEntriesCheckerHelperTest) - [Failed 0 of last 24](https://testray.liferay.com/home/-/testray/case_results/2257237054/history) could be related. I'm testing it.

**Comment by robertoDiaz** [comment]
ci:test:relevant

File: modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/scheduler/helper/AssetEntriesCheckerHelper.java [hunk 1]
```diff
@@ -291,6 +301,41 @@ private List<AssetEntry> _getAssetEntries(
...
+			AssetListEntry assetListEntry =
+				_assetListEntryLocalService.fetchAssetListEntry(
+					assetListEntryId);
+
+			if (assetListEntry == null) {
+				return null;
+			}
+
+			assetEntries = _assetListAssetEntryProvider.getAssetEntries(
+				assetListEntry, new long[] {SegmentsEntryConstants.ID_DEFAULT},
```
[the remark this answers is not in the document]
Comment (robertoDiaz) [reply modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/scheduler/helper/AssetEntriesCheckerHelper.java:326]: I'm analyzing it

**Comment by robertoDiaz** [comment]
ci:test:relevant

**Comment by robertoDiaz** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by robertoDiaz** [comment]
I've checked the FragmentEntry broken test and doesn't seem to be related with my changes.

My changes only affect the Asset Publisher notifications so I could not find any relation with this errors.

Is it right? If not, please tell me and I'll try to fix them.

Thank you in advance.

**Comment by YurenaC** [comment]
Hi @robertoDiaz 
I'll take a look at the failing test and will let you what I figure out :)

**Comment by YurenaC** [comment]
ci:test:relevant

**Comment by YurenaC** [comment]
ci:stop

**Comment by YurenaC** [comment]
Hi @robertoDiaz could you please rebase with master and run the relevant suite again?
I run a few failing test in my local machine after rebase and they passed.

Thanks!

**Comment by YurenaC** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
@robertoDiaz, tests are not related to your changes, they are happening in upstream too

**Comment by jkappler** [comment]
ci:stop

**Comment by jkappler** [comment]
ci:forward

**Comment by alessandromalves** [comment]
Hey @robertoDiaz I found a unique failure related to 
[PortalLogAssertorTest-modules-integration-mysql57-jdk8/0/8](https://testray.liferay.com/home/-/testray/case_results/2263177909). Could you please take a look at it? Thanks in advance!
cc.: @jkappler 


[Full log here](https://testray.liferay.com/reports/production/logs/2023-04/test-1-26/test-portal-acceptance-pullrequest(master)/8658/modules-integration-mysql57-jdk8/0/8/jenkins-console.txt.gz?authuser=0)
```[junit] Testcase: testScanXMLLog(com.liferay.portal.log.assertor.PortalLogAssertorTest):	FAILED
    [junit] 
    [junit] Portal log assert failure, see above log for more information: 
    [junit] null
    [junit] com.liferay.portal.verify.VerifyException: com.liferay.portal.kernel.exception.DuplicateRoleException: {roleId=0}
    [junit] 	at com.liferay.portal.verify.VerifyProcess.verify(VerifyProcess.java:84)
    [junit] 	at com.liferay.portal.verify.extender.internal.osgi.commands.VerifyProcessTrackerOSGiCommands._executeVerifyProcess(VerifyProcessTrackerOSGiCommands.java:317)
    [junit] 	at com.liferay.portal.verify.extender.internal.osgi.commands.VerifyProcessTrackerOSGiCommands.access$200(VerifyProcessTrackerOSGiCommands.java:72)
    [junit] 	at com.liferay.portal.verify.extender.internal.osgi.commands.VerifyProcessTrackerOSGiCommands$2.addingService(VerifyProcessTrackerOSGiCommands.java:229)

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:forward:force

**Pull request closed without merging**